rca/projects/models.py

```diff
@@ -218,6 +218,74 @@ class ProjectPage(BasePage):
         ]
     )
 
+    def _format_projects_for_gallery(self, projects):
+        """Internal method for formatting related projects to the correct
+        structure for the gallery template
+
+        Arguments:
+            projects: Queryset of project pages to format
+
+        Returns:
+            List: Maximum of 8 projects
+        """
+        items = []
+        for page in projects:
```

Comment:
Could this be `for page in projects[:8]:` so not all of the `projects` are processed if not needed?